content/news/052/index.md

```diff
@@ -368,6 +368,13 @@ _Discussions: [lobste.rs](https://lobste.rs/s/dsqumn/dioxus_labs_high_level_rust
 [leaving-post]: https://loglog.games/blog/leaving-rust-gamedev/
 
 
+### [Marquee Selection in Bevy]
+
+[@HeyZoos] published an [article] about building a Marquee Selection system in Bevy using `bevy_rapier` for collision detection.
```

Comment:
The link seems incorrect ?
```suggestion
[@HeyZoos] published an [article][Marquee Selection in Bevy] about building a Marquee Selection system in Bevy using `bevy_rapier` for collision detection.
```

To be noted it's a member only article, it's a risk to frustrate a lot of readers (myself included), I'd consider moving this news to a 1 liner, rather than a "title + paragraph" section ?